Bug stomp 6 21 (#123)
LaunchCodeEducation/intro-to-web-dev-curriculum: content/fetch/exercises/_index.md

   ```
   {{% /expand %}}

{{% notice blue Note "rocket" %}}

> You should not expect a response, take note of the error that you receive from the console!

Reviewer: ```suggestion
You should not expect a response. Take note of the error that you see in the console!
```
